server/events/command/scope_tags.go

```diff
@@ -0,0 +1,38 @@
+package command
+
+import (
+	"reflect"
+	"regexp"
+	"strings"
+)
+
+type ProjectScopeTags struct {
+	BaseRepo         string
+	PrNumber         string
+	Project          string
+	ProjectPath      string
+	TerraformVersion string
+	Workspace        string
+}
+
+func (s ProjectScopeTags) Loadtags() map[string]string {
```

Comment:
Should this be named like this?

```suggestion
func (s ProjectScopeTags) loadTags() map[string]string {
```

Reply:
Unfortunately I think we have to keep this public since it is used on `server/events/instrumented_project_command_runner.go`, maybe the name choice wasn't clear enough?

I guess that to make this private I could make an `ScopeTags` interface on the metrics package, and call the `loadTags` from there. Then `ProjectScopeTags` would be an implementation of that interface passed as argument to a function that load the tags on the scope. I'm not so sure this would work though, but I can give a try.

Reply:
That's ok, this was a nitpick.